ndex2/client.py

```diff
@@ -638,6 +796,47 @@ def get_network_summary(self, network_id):
         """
         Gets information about a network.
 
+        .. code-block:: python
+
+		from ndex2.nice_cx_network import NiceCXNetwork
+		import ndex2.client as nc
+		import ndex2
+		import networkx as nx
+		import pandas as pd
+		import os
+
+		new_summary=my_ndex.get_network_summary('INSERT NETWORK UUID HERE')
```

Comment:
Please add line showing how `my_ndex` is created. The only import needed is the one to import the Ndex2 client